DependencyInjection/PrestaSitemapExtension.php

```diff
@@ -43,6 +44,10 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('route_annotation_listener.xml');
         }
 
+//        if (class_exists(MessageHandlerInterface::class)) {
+            $loader->load('messenger.xml');
+//        }
+
```

Comment:
This condition fails because when running phpunit it returns false. Maybe it's better to make a boolean option for enabling messenger integration then?

Reply:
Maybe it is just because you used `class_exists` instead of `interface_exists` ?